include/ingescape_private.h

```diff
@@ -299,7 +299,8 @@ struct _igsagent_t {
     igs_core_context_t *context;
     char *services_channel;
     char *calls_channel; // DEPRECATED, replaced by services_channel
-
+    char *igs_channel;
```

Comment:
With what has been said on the other comments, if it turns out that this channel is only to exchange SPLIT messages, I suggest renaming it to `splits_channel`